### DISCUSSION TITLE
【寻人启事】近年，一名叫“牢大”的人失踪了

### DISCUSSION THREAD

**Discussion by tingsha1935** [share121/inter-knot#2250] [其他]
![d909364bde39b4bcb1b5243eed856d5](https://github.com/user-attachments/assets/0a49ddc5-c2e1-413a-a708-1ef09cf68497)
牢大，原名姓科，名比，口头禅：man。
近年，牢大热度很大，但迟迟未见牢大现身，不知何去何从，据说掉进了学名“曼巴空洞”的地方，一直了无音讯，希望绳匠出手救援，钱不是问题
奖励：丁尼*100000
           菲林*120
           加密母带*3

**Comment by YshengEs** [comment]
是的，当时我在现场，看见劳大做的直升机坠机进了曼巴空洞

**Comment by Genshin-open** [comment]
这个委托我接了

**Comment by suyaoguang** [comment]
我知道，我是目击者，当时牢大被直升机肘击进曼巴空洞了

**Comment by Genshin-Neuvillette** [comment]
劳大一肘就被肘进曼巴空洞里了，至今下落不明

**Comment by abassado** [comment]
man

**Comment by AetherDrop** [comment]
据说扣1能看到牢大↓

**Reply by laoxim2233** [reply]
1111111111

**Comment by HKFDBHZN** [comment]
1111111111111111

**Comment by peterSHI111** [comment]
man！！！！

**Comment by HeisenbergMrWhite** [comment]
![image](https://github.com/user-attachments/assets/ed943fb0-3ada-4a9e-8074-b01683f87b8e)
找到了，黑曼巴也在绳网里

**Comment by Orangered0801** [comment]
前几天还有一批绳匠进去了，听说以太活性读数高得离谱<img src="https://bbs-static.miyoushe.com/static/2024/03/21/438ac4c6210e00a8f94a3a1eb13eb55e_7655048688282650889.png" alt="丽都漫步-本" width="75" >

**Comment by FNN66** [comment]
这个委托我接了

**Reply by Usodaru48763** [reply]
還得是佐巴揚靠譜

**Comment by shenguojia123** [comment]
我当时就在现场，我亲眼看见老大和直升机都被无形的铁肘肘进了曼巴空洞里，那里以太活性读数高得离谱，是普通空洞的24倍

**Reply by shenguojia123** [reply]
可惜了，我的慧识就判出来两张黑桃
![7e43cbad-0c1d-414f-b2c2-291331336c1c](https://github.com/user-attachments/assets/7166c42b-aa42-466f-aca2-b4fdebcce176)

**Comment by deejoseph** [comment]
这个委托我接了

**Comment by GOTOYuanshen** [comment]
逆天

**Comment by 13346254991** [comment]
这个委托我接了

**Comment by Random114** [comment]
这个委托我接了

**Comment by kirinyao** [comment]
这个委托我接了

**Comment by Hitokui12138** [comment]
这个委托我接了!

**Comment by xiaomwm** [comment]
孩子们 这不好笑

**Comment by DKmsn** [comment]
这个委托我接了
